dev/cognitiveservices/data-plane/Language/analyzetext.json

```diff
@@ -864,6 +886,25 @@
         "documents"
       ]
     },
+    "CustomHealthcareResult": {
+      "type": "object",
+      "properties": {
+        "documents": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/HealthcareEntitiesDocumentResultWithDocumentDetectedLanguage"
```

Comment:
Is the custom model guaranteed to assign the same categories the prebuilt model assigns?

Reply:
Good point, if so, then the custom healthcare task will need to accept task parameters like fhirVersion and documentType

Reply:
You're right. Thanks for pointing this out.

Reply:
I meant to ask about these [categories](https://github.com/Azure/azure-rest-api-specs/blob/6c192122507728c8ad9eb999e7970262f45673c3/specification/cognitiveservices/data-plane/Language/preview/2022-10-01-preview/analyzetext.json#L925), are custom health models guaranteed to detect entities of these categories? I understand that the list of categories is modeled as extensible enum but I want to make sure documented ones will also be detected.

Reply:
@deyaaeldeen Yes, the custom API is guaranteed to return all prebuilt entities